results/final_results.json

```diff
@@ -15,9 +15,9 @@
         "standard": 16.43666922426724
     },
     "htmcore": {
-        "reward_low_FN_rate": 66.1922106936328,
-        "reward_low_FP_rate": 58.7930712907694,
-        "standard": 63.081419488725054
+        "reward_low_FN_rate": 60.852121191220256,
+        "reward_low_FP_rate": 45.428862226866734,
+        "standard": 55.50231971786488
```

Comment:
Out of interest, @steinroe on your changes, I get slightly different results (seed is fixed AFAIK) 

```
Running score normalization step
Final score for 'htmcore' detector on 'standard' profile = 58.90
Final score for 'htmcore' detector on 'reward_low_FP_rate' profile = 48.80
Final score for 'htmcore' detector on 'reward_low_FN_rate' profile = 64.27
```

Guess I have just luckier HW :smirk:

Reply:
Haha even better! Although I don't understand why when the seeds are fixed.